jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamodb/editor/IndexRenderer.kt

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.dynamodb.editor
+
+import com.intellij.ui.SeparatorWithText
+import com.intellij.ui.SimpleListCellRenderer
+import com.intellij.util.ui.UIUtil
+import com.intellij.util.ui.components.BorderLayoutPanel
+import software.aws.toolkits.jetbrains.services.dynamodb.Index
+import java.awt.Component
+import java.awt.Font
+import javax.swing.JComponent
+import javax.swing.JList
+import javax.swing.ListCellRenderer
+import javax.swing.border.Border
+
+class IndexRenderer(private val model: IndexComboBoxModel) : ListCellRenderer<Index?> {
+    private val rowRenderer = SimpleListCellRenderer.create<Index>("") { it.displayName }
+    private val separator = SeparatorWithText().apply {
+        textForeground = UIUtil.getListForeground()
+        font = font.deriveFont(Font.PLAIN)
```

Comment:
What does this change?

Reply:
It is bold by default

Reply:
<img width="1054" alt="Screen Shot 2021-05-14 at 2 32 33 PM" src="https://user-images.githubusercontent.com/8992246/118333808-3e30b780-b4c1-11eb-968b-b35c642da48b.png">